src/contexts/Localization/helpers.ts

```diff
@@ -0,0 +1,21 @@
+import { EN, LanguageCode } from '../../config/localization/languages'
+
+const publicUrl = process.env.PUBLIC_URL
+
+export const LS_KEY = 'pancakeswap_lang'
+
+export const fetchLocale = async (code: LanguageCode) => {
+  const response = await fetch(`${publicUrl}/locales/${code}.json`)
```

Comment:
I'm still slightly confused as to how `translations.json` is populated and maintained, and then where the per-language translations are stored? Will we also have `.json` files for each language in the codebase, or are the non-EN translations fetched remotely?

Reply:
Magic Hutch. Magic.

Reply:
Once the integration is complete it will all become clear.